project.clj

```diff
@@ -12,7 +12,8 @@
                  [hickory "0.7.1"]
                  [com.cognitect/transit-cljs "0.8.248"]
                  [status-im/pluto "iteration-2-SNAPSHOT"]
-                 [mvxcvi/alphabase "1.0.0"]]
+                 [mvxcvi/alphabase "1.0.0"]
+                 [rasom/cljs-react-navigation "0.1.4"]]
```

Comment:
Move to status repo

Reply:
though this is about uberjar... let's fix this in a separate PR